src/main/java/com/connexta/search/common/configs/SolrConfiguration.java

```diff
@@ -22,8 +22,10 @@
   public static final String SOLR_COLLECTION = "searchTerms";
   public static final String CONTENTS_ATTRIBUTE_NAME = "contents";
   public static final String ID_ATTRIBUTE_NAME = "id";
+  public static final String MEDIA_TYPE_ATTRIBUTE_NAME = "media.type";
```

Comment:
✏️ `media_type` or `mediaType` or `mediatype` to eliminate the need to wrap in quotes

Reply:
I used Susan's CST spreadsheet for the name of the term. If you are comfortable departing from the attribute names she uses, I'll change it to something that doesn't need double quotes.

Reply:
I think we can depart from that list👍

Reply:
I think it would be a good bet to use the format Solr specifies for field names: `media_type`. It's not enforced, but at least it would be consistent. https://lucene.apache.org/solr/guide/6_6/defining-fields.html

Reply:
I'll do that. It's an arbitrary convention, but a a convention is better than no convention. 

I don't believe that Solr will always be the only MIS search engine or database. When additional resources are added to our data layer, we should still be able to stick with the underscore convention, even if it isn't necessarily the convention for those databases/search engines.